apps/src/code-studio/pd/workshop_dashboard/reports/local_summer_workshop_daily_survey/results_loader.jsx

```diff
@@ -0,0 +1,53 @@
+import React, {PropTypes} from 'react';
+import $ from "jquery";
+import Spinner from '../../../components/spinner';
+import Results from './results';
+import _ from 'lodash';
+
+export class ResultsLoader extends React.Component {
+  static propTypes = {
+    params: PropTypes.shape({
+      workshopId: PropTypes.string.isRequired
+    })
+  };
+
+  state = {loading: true};
+
+  componentDidMount() {
+    this.load();
+  }
+
+  load() {
+    const url = `/api/v1/pd/workshops/${this.props.params['workshopId']}/local_workshop_daily_survey_report`;
+
+    this.loadRequest = $.ajax({
+      method: 'GET',
+      url: url,
+      dataType: 'json'
+    }).done(data => {
+      this.setState({
+        loading: false,
+        questions: data['questions'],
+        thisWorkshop: data['this_workshop'],
+        allMyWorkshops: data['all_my_workshops'],
+        sessions: Object.keys(data['this_workshop'])
+      });
+    });
+  }
+
+  render() {
+    if (this.state.loading) {
+      return (
+        <div>
+          <Spinner/>
+        </div>
+      );
+    } else {
+      return (
+        <Results
+          {..._.omit(this.state, ['loading'])}
```

Comment:
note, this omit can be done in raw ES6 by adding the following to the beginning of the function:
```jsx
const {loading, ...data} = this.state;
// Now we have 2 local vars, loading and data, where `data` contains all the state except `loading`
```